R/create_inclusion_dates.R

```diff
@@ -0,0 +1,62 @@
+#' Create inclusion dates from all the inclusion events
+#'
+#' This function takes the output from
+#' [join_inclusions()] and defines the final inclusion dates, raw and stable
+#' based on all inclusion event types. Since inclusion requires at least two
+#' events (of any type), this function keeps only those with 2 or more events.
```

Comment:
Just to spell this out, since it confuses people again and again (and I think this description is probably the best I've come up with so far - or what do you think?)
```suggestion
#' events (can be a repetition of the same type of inclusion event or any combination of different types of inclusion events), this function keeps only those with 2 or more events.
```

Reply:
Seems like a very good idea to be very explicit about it here if this is a step that confuses people :) 

How about this instead, @Aastedet? I avoid the word "repetition" here bc it *could* maybe be misunderstood as being a repetition of the actual same event not just the same type.

```suggestion
#' events (can be multiple events of the same type or any combination of different types), this function keeps only those with 2 or more events.
```